holoviews/core/sheetcoords.py

```diff
@@ -517,7 +517,6 @@ def _boundsspec2slicespec(boundsspec,scs):
 
         l_idx = int(np.ceil(l_m-0.5))
         t_idx = int(np.ceil(t_m-0.5))
-        # CBENHANCEMENT: Python 2.6's math.trunc()?
         r_idx = int(np.floor(r_m+0.5))
```

Comment:
Out of scope for this PR. But because these are scalars, it is significantly faster* to use the built-in math module:

![image](https://github.com/holoviz/holoviews/assets/19758978/d8830700-c87f-412b-bfb3-3bd4a9e276ec)

*) Only relative - absolute it does not make a difference :upside_down_face: